### PR TITLE
Add PHP 8.2 Support

### PR DESCRIPTION
**Pull request by patinthehat**
# Summary

This PR adds PHP 8.2 to the tests workflow.

_id:php82-support/v1_

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
             fail-fast: true
             matrix:
                 os: [ubuntu-latest]
-                php: [8.1, 8.0, 7.4]
+                php: [8.2, 8.1, 8.0, 7.4]
```
Comment (sanmai) [inline .github/workflows/run-tests.yml:12]:
```suggestion
                php: ['8.2', '8.1', '8.0', '7.4']
```

Comment (patinthehat) [reply .github/workflows/run-tests.yml:12]: @sanmai Most of the other Spatie workflows are not quoted, so just curious - why this one? :thinking:

Comment (sanmai) [reply .github/workflows/run-tests.yml:12]: - `shivammathur/setup-php` [expects to see strings](https://github.com/shivammathur/setup-php#php-version-required) for `php-version`.
- There was an issue when it installed the really latest version if you didn't specify a string. I.e. for `8.0` it used to install `8.1.x.` and for `'8.0'` it used to install `8.0.x`. But it looks like it is no longer an issue.
- Lastly, it looks prettier in the log:

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**